docs/walkthroughs/developing-apbs-locally.adoc

```diff
@@ -161,6 +174,29 @@ metadata:
 ...
 ----
 
+=== Bind Annotations
+If your Mobile Service can be bound to a Mobile Client, it is possible to annotate the Mobile Client during the bind step of the APB.
+This allows specific information about the Mobile Service (for example, links to the Service) to be displayed in the UI within the context of the Mobile Client.
+
+image:./images/service-details.png[Mobile Service Details]
+
+The annotation should stick to the following format: 
+
+[source,yaml]
+----
+org.aerogear.binding.<service-instance-name>/<key>: <value>
+----
+
+The value is a JSON string with the structure:
+
+[source,json]
+----
+{"label": "<display-name>", "type": "[string|href]", "value": "<value>"}
+----
+
+An example of this can be seen in the Keycloak APB, link:https://github.com/aerogearcatalog/keycloak-apb/blob/master/roles/bind-keycloak-apb/tasks/main.yml#L107-L119[here]. These annotations should be link:https://github.com/aerogearcatalog/keycloak-apb/blob/master/roles/unbind-keycloak-apb/tasks/main.yml#L48-L60[removed on unbind]. 
```

Comment:
Can you make this link to a commit hash, so it doesn't become invalid when master is updated, like this:
`https://github.com/aerogearcatalog/keycloak-apb/blob/4d160fd4724b68994ec0fe2ed7952930b2ef3d9e/roles/bind-keycloak-apb/tasks/main.yml#L107-L119`

Reply:
Good point, thanks :+1: